CMakeLists.txt

```diff
@@ -202,8 +202,13 @@ if(WIN32)
     set(socketlistener_c_files
         ./src/socket_listener_win32.c
     )
+elseif(LINUX)
+    set(socketlistener_c_files
+        ./src/socket_listener_berkeley.c
+    )
 else()
-    set(socketlistener_c_files)
+    set(socketlistener_c_files
+    )
```

Comment:
Yep.

---
In reply to: [109059639](https://github.com/Azure/azure-uamqp-c/pull/69#discussion_r109059639) [](ancestors = 109059639)